data/outfits/god.txt

```diff
@@ -19,18 +17,80 @@ outfit "Ruin-The-Fun God Mode"
 	"corrosion protection" 1000000
 	"leak protection" 1000000
 	"burn protection" 1000000
-	bunks 1000000
 	description "Absolutely illegal."
 
+outfit "Ruin-The-Fun God Jammer"
+	"display name" "RTF God Jammer"
+	category "Special"
+	cost 0
+	thumbnail "outfit/large radar jammer"
+	"radar jamming" 1000000
+	"optical jamming" 1000000
+	"scan interference" 1000000
+	description "Allows you to jam a lot."
+
+outfit "Ruin-The-Fun God Scanner"
+	"display name" "RTF God Scanner"
+	category "Special"
+	cost 0
+	thumbnail "outfit/outfit scanner"
+	"tactical scan power" 1000000
+	"asteroid scan power" 1000000
+	"outfit scan power" 1000000
+	"cargo scan power" 1000000
+	"tactical scan efficiency" 1000000
+	"asteroid scan efficiency" 1000000
+	"outfit scan efficiency" 1000000
+	"cargo scan efficiency" 1000000
+	description "Allows you to scan far and wide."
+
+outfit "Ruin-The-Fun God Generator"
+	"display name" "RTF God Generator"
+	category "Special"
+	cost 0
+	thumbnail "outfit/fusion"
+	"energy generation" 1000000
+	"solar collection" 1000000
+	description "Allows you to generate a lot of power."
 
+outfit "Ruin-The-Fun God Engine"
+	"display name" "RTF God Engine"
+	category "Special"
+	cost 0
+	thumbnail "outfit/tiny atomic thruster"
+	"thrust" 200
+	"reverse thrust" 200
+	description "Allows you to run really fast."
+
+outfit "Ruin-The-Fun God Steering"
+	"display name" "RTF God Steering"
+	category "Special"
+	cost 0
+	thumbnail "outfit/tiny atomic steering"
+	"turn" 5000
+	description "Allows you to turn really fast."
```

Comment:
I was trying to get slightly stronger than A865 which has `turn` of 3597. If you have more than 3 on a small ship then it will became too difficult to control. I need to test if lower it to 4000 would work on small ships.

Reply:
5000 works fine for big ships, but is too fast for small ships.

Perhaps using playing with attributes `drag`, `inertia reduction`, `mass`, `inertia reduction` could help making an engine that gives the same thrust for all ships?

For instance it gives a lot of thrust but also some mass, it would give similar speed to both light and heavy ships. But that could also have side effects...

Reply:
What if the steering/engine increases the mass of the ship by a high amount, and then adds the turning/thrust adequate to move a ship with that mass? If the added mass is high enough, the previous ship's mass would be negligible, and any ship with that engine would move the same.

Reply:
With `mass 100000 ` the acceleration and turn is down to a reasonable range for the few ships I tested.